ToJ set updates + artist credit fixes

kayorga opened the pull request.

kayorga committed:
card updates; rarity restructuring; fixed artist credits

kayorga committed:
Merge remote-tracking branch 'upstream/development' into draft

kayorga committed:
rarity revamp + flavor review

kayorga committed:
card fixes

kayorga committed:
hot pie buff

kayorga committed:
hot pie update

kayorga committed:
Merge remote-tracking branch 'upstream/development' into draft

kayorga committed:
MaV and fixes

kayorga committed:
illustrator credits + fixes

kayorga committed:
Merge remote-tracking branch 'upstream/development' into draft

kayorga committed:
more fixes; flavor text; artist credits

stopfstedt commented on packs/ToJ.json:
please add a release date here and change `workInProgress` to `false` when done.

stopfstedt reviewed (commented):
i'm [ready from my end](https://github.com/ThronesDB/thronesdb/pull/958) to go live with this, i rolled the code-changes to support this set out to TDB just now.

the set needs a publication date and the `workInProgress` flag toggled to `false`, otherwise these card will show up as "work in progress" on the site.

some of the cards, like the one below, are still using the placeholder/auto-generated images. 

<img width="1393" height="506" alt="image" src="https://github.com/user-attachments/assets/af06b1b1-7bd6-4e15-90da-a0df2dc3bae3" />


also, that `draftmancer/ToJ.txt`  file - is this needed in this repo? if not then please delete.

stopfstedt reviewed (changes requested):
see comments above.

kayorga committed:
fixes

kayorga committed:
fixes

Patane97 committed:
Adding JfE.json (#191)

Patane97 committed:
Fixing formatting on JfE (#192)

* Fixing formatting on JfE

* Fixing LotW as well whoops

kayorga committed:
internally hosted images

kayorga committed:
Merge remote-tracking branch 'upstream/development' into draft

kayorga committed:
updated draftmancer file with internally hosted images

stopfstedt committed:
corrects date format. (#194)

jimorie committed:
Change Tyrion Lannister (VDS) to unique (#189)

kayorga committed:
fixed some artist credits (#193)

Patane97 committed:
Releasing NCbT & other fixes (#196)

* Adding NCbT

* Fixing various flavor and \r inconsistencies from recent packs

Patane97 committed:
Whoops! (#197)

kayorga committed:
2026 edition fixes

kayorga committed:
trait fix

kayorga committed:
draft starter deck limits

stopfstedt committed:
use rarity codes instead of display labels for rarity values.

the ThronesDB cards importer is expecting these codes to be provided.

stopfstedt committed:
add rarity as optional enum to card schema definition.

this will catch any typos in rarity values when running the `npm run
validate`.

kayorga committed:
Merge pull request #1 from stopfstedt/draft_fixes

Draft fixes

Patane97 reviewed (approved):
The changes seem good from my perspective with TheIronThrone.

@kayorga Just mentioning where this should be merged into; as the intent is for us to bring this to the live site (ie. `workInProgress: false`), I would recommend this actually merge into `throneteki:master`, rather than playtesting.

The changes from there will then (eventually) flow into `development`; I will handle the merge conflicts (that is an issue I have accepted with the playtesting -> live workflow for a while now, this would be no exception).

kayorga committed:
fixed hot pie uniqueness

Patane97 committed:
Merge branch 'master' into draft

kayorga committed:
removed playtesting stuff
